emcc.py

```diff
@@ -2440,6 +2444,14 @@ def check_bad_eq(arg):
       options.llvm_lto = int(newargs[i + 1])
       newargs[i] = ''
       newargs[i + 1] = ''
+    elif newargs[i].startswith('--closure-args'):
+      check_bad_eq(newargs[i])
+      args = newargs[i + 1]
+      if (args[0] == '"' and args[-1] == '"') or (args[0] == "'" and args[-1] == "'"):
+        args = args[1:-1]
```

Comment:
I doubt it's necessary to care about the quotation marks. If I type `"foo bar"` in the terminal, the quotation marks should be stripped by the shell and the string in sys.argv should be `foo bar`, isn't it?

Reply:
It's not necessary, but idea was to be more robust with command line and response file inputs. Probably excessive, dropped.